packages/builder/src/components/automation/AutomationBuilder/FlowChart/ActionModal.svelte

```diff
@@ -19,10 +19,12 @@
   export let lastStep
 
   let syncAutomationsEnabled = $licensing.syncAutomationsEnabled
+  let triggerAutomationsEnabled = true
```

Comment:
what does the boolean do - I don't see anywhere that changes it?

Reply:
I had it set to true for testing, fixed.